isso/views/comments.py

```diff
@@ -275,7 +275,7 @@ def new(self, environ, request, uri):
             data["website"] = normalize(data["website"])
 
         data['mode'] = 2 if self.moderated else 1
-        data['remote_addr'] = utils.anonymize(str(request.remote_addr))
+        data['remote_addr'] = utils.anonymize(str(request.access_route[0]))  # Using .access_route instead of .remote_addr to take into account HTTP_X_FORWARDED_FOR header
```

Comment:
Wouldn't this allow users to vote multiple times by spoofing the X-Forwarded-For header?

Reply:
Replies were made in the PR discussion